{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "datePublic": "2011-03-14T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "feedparser.py in Universal Feed Parser (aka feedparser or python-feedparser) before 5.0.1 allows remote attackers to cause a denial of service (application crash) via a malformed DOCTYPE declaration."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2011-04-21T09:00:00",
                "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
                "shortName": "redhat"
            },
            "references": [
                {
                    "name": "[oss-security] 20110314 CVE request for python-feedparser",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "http://openwall.com/lists/oss-security/2011/03/14/18"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "http://support.novell.com/security/cve/CVE-2011-1156.html"
                },
                {
                    "name": "43730",
                    "tags": [
                        "third-party-advisory",
                        "x_refsource_SECUNIA"
                    ],
                    "url": "http://secunia.com/advisories/43730"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://code.google.com/p/feedparser/issues/detail?id=91"
                },
                {
                    "name": "46867",
                    "tags": [
                        "vdb-entry",
                        "x_refsource_BID"
                    ],
                    "url": "http://www.securityfocus.com/bid/46867"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=684877"
                },
                {
                    "name": "44074",
                    "tags": [
                        "third-party-advisory",
                        "x_refsource_SECUNIA"
                    ],
                    "url": "http://secunia.com/advisories/44074"
                },
                {
                    "name": "[oss-security] 20110315 Re: CVE request for python-feedparser",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "http://openwall.com/lists/oss-security/2011/03/15/11"
                },
                {
                    "name": "[opensuse-updates] 20110408 openSUSE-SU-2011:0314-1 (moderate): python-feedparser security update",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "http://lists.opensuse.org/opensuse-updates/2011-04/msg00026.html"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://bugzilla.novell.com/show_bug.cgi?id=680074"
                },
                {
                    "name": "MDVSA-2011:082",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_MANDRIVA"
                    ],
                    "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2011:082"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "secalert@redhat.com",
                    "ID": "CVE-2011-1156",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "feedparser.py in Universal Feed Parser (aka feedparser or python-feedparser) before 5.0.1 allows remote attackers to cause a denial of service (application crash) via a malformed DOCTYPE declaration."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "[oss-security] 20110314 CVE request for python-feedparser",
                            "refsource": "MLIST",
                            "url": "http://openwall.com/lists/oss-security/2011/03/14/18"
                        },
                        {
                            "name": "http://support.novell.com/security/cve/CVE-2011-1156.html",
                            "refsource": "CONFIRM",
                            "url": "http://support.novell.com/security/cve/CVE-2011-1156.html"
                        },
                        {
                            "name": "43730",
                            "refsource": "SECUNIA",
                            "url": "http://secunia.com/advisories/43730"
                        },
                        {
                            "name": "https://code.google.com/p/feedparser/issues/detail?id=91",
                            "refsource": "CONFIRM",
                            "url": "https://code.google.com/p/feedparser/issues/detail?id=91"
                        },
                        {
                            "name": "46867",
                            "refsource": "BID",
                            "url": "http://www.securityfocus.com/bid/46867"
                        },
                        {
                            "name": "https://bugzilla.redhat.com/show_bug.cgi?id=684877",
                            "refsource": "CONFIRM",
                            "url": "https://bugzilla.redhat.com/show_bug.cgi?id=684877"
                        },
                        {
                            "name": "44074",
                            "refsource": "SECUNIA",
                            "url": "http://secunia.com/advisories/44074"
                        },
                        {
                            "name": "[oss-security] 20110315 Re: CVE request for python-feedparser",
                            "refsource": "MLIST",
                            "url": "http://openwall.com/lists/oss-security/2011/03/15/11"
                        },
                        {
                            "name": "[opensuse-updates] 20110408 openSUSE-SU-2011:0314-1 (moderate): python-feedparser security update",
                            "refsource": "MLIST",
                            "url": "http://lists.opensuse.org/opensuse-updates/2011-04/msg00026.html"
                        },
                        {
                            "name": "https://bugzilla.novell.com/show_bug.cgi?id=680074",
                            "refsource": "CONFIRM",
                            "url": "https://bugzilla.novell.com/show_bug.cgi?id=680074"
                        },
                        {
                            "name": "MDVSA-2011:082",
                            "refsource": "MANDRIVA",
                            "url": "http://www.mandriva.com/security/advisories?name=MDVSA-2011:082"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "assignerShortName": "redhat",
        "cveId": "CVE-2011-1156",
        "datePublished": "2011-04-11T18:00:00",
        "dateReserved": "2011-03-03T00:00:00",
        "dateUpdated": "2011-04-21T09:00:00",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}